service/src/test/java/dk/dbc/rr/oai/OaiBeanIT.java

```diff
@@ -63,5 +68,307 @@ public void testGetAllowedSets() throws Exception {
         assertThat(an.stream().sorted().collect(toList()), is(asList("art", "nat")));
         Set<String> abno = oaiBean.getAllowedSets(ABNO_TRIPLE, "127.0.0.1");
         assertThat(abno.stream().sorted().collect(toList()), is(asList("art", "bkm", "nat", "onl")));
+        Set<String> anonymous = oaiBean.getAllowedSets(null, "8.8.8.8");
+        assertThat(anonymous.stream().sorted().collect(toList()), is(asList("art", "nat")));
+    }
+
+    @Test(timeout = 2_000L)
```

Comment:
Kan man evt bruge default timeout, så man ikke behøver timeout på alle @Test?
(Jeg ved dog ikke hvordan man overruler værdien, hvis man har brug for forskellige værdier i nogle tests)
https://github.com/junit-team/junit4/wiki/Timeout-for-tests
@Rule
public Timeout globalTimeout = Timeout.seconds(10); // 10 seconds max per method tested